lua/autorun/client/cl_autoclean.lua

```diff
@@ -8,11 +8,8 @@ local function notifyPlayer( notification )
     print( message )
 end
 
-net.Receive( "CFC_AutoClean_RunCommand", function() 
-    local command = net.ReadString()
-
-    if isAllowedCommand[command] then
-        notifyPlayer( "The server is running '" .. command .. "' on your client.." )
+net.Receive( "CFC_RunAutoClean", function() 
+    for command, _ in pairs(cleanupCommands) do
```

Comment:
would an array of commands be better now because the true value doesn't change anything?

Reply:
what does the loop look like for running through an array again?

Reply:
```lua
for _, k in pairs(r) do

end
```

Reply:
or
```lua
for i=1, #r do
    local k = r[i]
end
```